ansible/roles/consul/templates/config.json.j2

```diff
@@ -0,0 +1,30 @@
+{# this template is used to generate a config.json for booting a consul server or a consul agent, based on the given environment inventory #}
+{
+{% if 'consul_servers' in group_names %}
+    "server": true,
+    "bootstrap_expect": {{ groups['consul_servers']|length }},
+{% endif %}
+    {% for host in groups['consul_servers'] %}
+{% if loop.first -%}
+    "start_join": ["{{host}}"
+{%- endif %}
+{% if loop.last and loop.length == 1 -%}
```

Comment:
@lehoanganh I see what you changed here. this did not work before if we had only one host. even more ugly now :-) but working.